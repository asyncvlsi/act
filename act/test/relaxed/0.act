template <pint x | pint y>
defproc foo (bool a, b[y])
{
  bool c[x];
  bool d[y];

  prs {
   a => b-
  }
}


foo<4> f[3] @ [type=0];
