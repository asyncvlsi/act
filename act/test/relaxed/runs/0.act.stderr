ERROR: File `0.act', line 2, col 27
	Expressions in port parameter list can only use strict template parameters
